module-5/09_11_2024_M05W03/README.md

```
@@ -0,0 +1,124 @@
+
+    - $\eta$ là learning rate
+    - $\frac{\partial L}{\partial W^{[l]}}$ là gradient của loss với $W^{[l]}$
+    - $W^{[l]}$ là ma trận trọng số của layer $l$
+
+### 4.3 Activation Function
+- Là các hàm toán học được áp dụng cho đầu ra của các nơ-ron trong MLP, với ý nghĩa chính là `giúp MLP đưa ra quyết định có nên kích hoạt nơ-ron hay không`. 
+- Nó giúp chuyển đổi đầu ra tuyến tính của một lớp thành một dạng phi tuyến, cho phép mạng nơ-ron học được các quan hệ phức tạp hơn.
+![Activation function](./assets/images/activation_functions.png)
+
+- Các hàm kích hoạt thường được sử dụng:
+    - Sigmoid:
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)